src/content/components/FeatureView/FeatureView.js

```diff
@@ -268,6 +269,7 @@ export class FeatureView extends React.PureComponent {
       heading={<a href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${metaId}`}>{metaDisplayName}</a>}
       fileBug={`blocked=${metaId}`}
       subHeading={<React.Fragment>This list includes bugs in any component blocking meta bug <a href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${metaId}`}> {metaId}</a> <CopyButton text={metaId} title="Copy bug number" /></React.Fragment>}>
+      <CompletionBar bugs={this.state.bugs} startDate={this.state.start} endDate={this.state.due} />
```

Comment:
you can remove `startDate={this.state.start} endDate={this.state.due} ` since they are empty anyway

Reply:
oh of course!